src/wallet/wallet.cpp

```diff
@@ -4998,6 +5074,11 @@ void CWalletCert::GetAmounts(std::list<COutputEntry>& listReceived, std::list<CO
     }
 }
 
+bool CWalletCert::IsTrusted(bool /*unused*/) const 
+{
+    return IsTrusted();
+}
+
```

Comment:
By conflating the two IsTrusted functions into one, we can avoid this pass-through